Kitodo/src/main/java/org/goobi/production/cli/helper/CopyProcess.java

```diff
@@ -100,7 +100,7 @@ public String prepare(ImportObject io) {
             this.myRdf = UghImplementation.INSTANCE.createMetsMods(myPrefs);
             this.myRdf.read(this.metadataFile.getPath());
         } catch (PreferencesException | ReadException e) {
-            logger.error(e);
+            logger.error(e.getMessage(), e);
```

Comment:
set ErrorMessage

Reply:
It looks like in this place there should no message be generated onto the user screen. How can I use setErrorMessage when there should no error message be generated? (Same is true for the other remarks of the same category.)

Reply:
True, but in most of the cases catching exception before it can be displayed do user is not good idea.

Reply:
There must be reasons for it why it was implemented this way. This commit is not to check if this is good or bad, it is only to make sure the stack trace is logged. Examining reasons is a separate task. You can create a ticket for this if you find it is necessary.